db/migrate/20151118174316_create_bi_visualizations_and_bi_datasets.rb

```
@@ -0,0 +1,29 @@
+Sequel.migration do
+  up do
+    Rails::Sequel::connection.run 'CREATE EXTENSION IF NOT EXISTS "uuid-ossp"'
```

just thinking aloud: this is in previous migrations, I'm not sure this is the expected practice, being defensive in this short of thing. The alternative is fail fast if you haven't successfully applied previous migrations